docs/html/quickstart.rst

```diff
@@ -2,90 +2,126 @@
+
 
+3. Set up a virtual environment
+-------------------------------
 
-Install a package that's already been downloaded from `PyPI`_ or
-obtained from elsewhere. This is useful if the target machine does not have a
-network connection:
+Before using pip, we recommend you set up and use a virtual environment to
+isolate your project packages.
+
+See `Creating Virtual Environments`_ for more information.
+
+Common tasks
+============
+
+Install a package from `PyPI`_
+------------------------------
 
 .. tab:: Unix/macOS
 
    .. code-block:: console
 
-      $ python -m pip install SomePackage-1.0-py2.py3-none-any.whl
+      $ python -m pip install SomePackage
       [...]
       Successfully installed SomePackage
 
 .. tab:: Windows
 
    .. code-block:: console
 
-      C:\> py -m pip install SomePackage-1.0-py2.py3-none-any.whl
+      C:\> py -m pip install SomePackage
       [...]
       Successfully installed SomePackage
 
-Show what files were installed:
+Install a package from GitHub
+------------------------------
 
-.. tab:: Unix/macOS
+Pip can install packages from common version control systems (VCS), including
+GitHub.
 
-   .. code-block:: console
+For example, to install a specific commit from the Django project, run:
 
-      $ python -m pip show --files SomePackage
-      Name: SomePackage
-      Version: 1.0
-      Location: /my/env/lib/pythonx.x/site-packages
-      Files:
-      ../somepackage/__init__.py
-      [...]
+.. code-block:: console
 
-.. tab:: Windows
+  pip install git+https://github.com/django/django.git@45dfb3641aa4d9828a7c5448d11aa67c7cbd7966
 
-   .. code-block:: console
+See VCS support for more information.
 
-      C:\> py -m pip show --files SomePackage
-      Name: SomePackage
-      Version: 1.0
-      Location: /my/env/lib/pythonx.x/site-packages
-      Files:
-      ../somepackage/__init__.py
-      [...]
+Install a package you have already downloaded
+---------------------------------------------
 
-List what packages are outdated:
+This is useful if the target machine does not have a network connection:
 
 .. tab:: Unix/macOS
 
    .. code-block:: console
 
-      $ python -m pip list --outdated
-      SomePackage (Current: 1.0 Latest: 2.0)
+      $ python -m pip install SomePackage-1.0-py2.py3-none-any.whl
+      [...]
+      Successfully installed SomePackage
 
 .. tab:: Windows
 
    .. code-block:: console
 
-      C:\> py -m pip list --outdated
-      SomePackage (Current: 1.0 Latest: 2.0)
+      C:\> py -m pip install SomePackage-1.0-py2.py3-none-any.whl
+      [...]
+      Successfully installed SomePackage
+
+Install packages from a file
+----------------------------
 
-Upgrade a package:
+Many Python projects use a requirements.txt file to specify the list of packages
+that need to be installed for the project to run. To install the packages
+listed in the file, run:
+
+.. code-block:: console
+
+  pip install -r requirements.txt
+
+See requirements files for more information.
```

Comment:
Again, I;d like to link the requirements section in the user guide here, but I don't know how